src/components/Utils/order/showUpdateOrderError.js

```diff
@@ -9,10 +11,29 @@ export const ERROR_CODES = {
   orderOpen: 'orderOpen',
   orderClosed: 'orderClosed',
   accessProviderNotFound: 'accessProviderNotFound',
+  missingInstanceStatus: 'missingInstanceStatus',
+  missingInstanceType: 'missingInstanceType',
+  missingLoanType: 'missingLoanType',
+
 };
 
 const POL_NUMBER_KEY = 'poLineNumber';
 
+const getCallout = (callout, code, error, path) => {
```

Comment:
so it doesn't `get` anything, right? Maybe `showMessage` or `showNotification` is a more proper name, what do you think?